lib/net/Makefile

```diff
@@ -5,20 +5,33 @@ NETIFFILES := $(filter-out $(LWIPDIR)/netif/slipif.c,$(NETIFFILES))
 LWIPSRCS := $(COREFILES) \
             $(CORE4FILES) \
             $(CORE6FILES) \
-            $(NETIFFILES) \
-            $(APIFILES)
+            $(APIFILES) \
+            $(NETIFFILES)
 
 # Include driver sources
 DRIVERSRCS := $(NXDK_DIR)/lib/net/pktdrv/pktdrv.c \
               $(NXDK_DIR)/lib/net/nforceif/src/driver.c \
               $(NXDK_DIR)/lib/net/nforceif/src/sys_arch.c
 
-SRCS += $(LWIPSRCS) $(DRIVERSRCS)
+NET_SRCS := \
```

Comment:
*(This is a generic comment about PR description, but made on code so we can resolve it more easily)*

> - Why does the net-Makefile has extra `CXXFLAGS` while other libraries don't? Can't find the reason 🙈😅

Because someone made a C++ project and wanted to use these libs + these headers are probably C++ aware already.